src/Microsoft.IdentityModel.Aad/ClaimsAccessorsDoc.md

```diff
@@ -0,0 +1,90 @@
+﻿
+# Claims
+
+Type | Accessor | Used for authorization | Privacy | Claims
+--   | --       | --                     | --      | --     
+string | GetActor | False |  | actort<BR/> http://schemas.xmlsoap.org/ws/2009/09/identity/claims/actor
+string | GetIsGuestAccount | True |  | acct
+string | GetLastAuthenticatedTime | False |  | auth_time
+string | GetBirthdate | False | EUII | birthdate<BR/> http://schemas.xmlsoap.org/ws/2005/05/identity/claims/dateofbirth
+string | GetCountry | False | EUII | ctry
+string | GetEmail | False | EUII | email<BR/> http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress
+string | GetFamilyName | False | EUII | family_name<BR/> http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname
+string | GetGender | False | EUII | gender<BR/> http://schemas.xmlsoap.org/ws/2005/05/identity/claims/gender
+string | GetGivenName | False | EUII | given_name<BR/> http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname
+string | GetNameId | False |  | nameid<BR/> http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier
+string | GetSubject | False |  | sub<BR/> http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier
+string | GetWebsite | False |  | website<BR/> http://schemas.xmlsoap.org/ws/2005/05/identity/claims/webpage
+string | GetUniqueName | False | EUPI | unique_name<BR/> http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name
+string | GetObjectId | True | EUPI | oid<BR/> http://schemas.microsoft.com/identity/claims/objectidentifier
+IEnumerable<string> | GetScopes | True |  | scp<BR/> http://schemas.microsoft.com/identity/claims/scope
+string | GetTenantId | True | OII | tid<BR/> http://schemas.microsoft.com/identity/claims/tenantid
+string | GetAuthenticationContextClassReference | False |  | acr<BR/> http://schemas.microsoft.com/claims/authnclassreference
+string | GetAdfs1Email | False | EUPI | adfs1email<BR/> http://schemas.xmlsoap.org/claims/EmailAddress
+string | GetAdfs1Upn | False | EUPI | adfs1upn<BR/> http://schemas.xmlsoap.org/claims/UPN
+string | GetAmr | False |  | amr<BR/> http://schemas.microsoft.com/claims/authnmethodsreferences
```

Comment:
amr is definitely used in Authorization as this defines how the subject authenticated with the identity provider.